GraphQL-TSC.md

```diff
@@ -169,19 +169,19 @@ Because we work in a distributed environment, the voting process must account fo
 * A notice is sent via email that the vote will conclude in three business days, reminding those who haven't voted that they should do so. The vote will conclude at the end of this time.
 * The election results will not change if additional TSC members vote.
 
-#### Ensuring efficient votes
+#### TSC voting member eligibility
 
-Should a TSC member miss three or more of the prior five meetings, they will no longer vote or be counted toward the total number of voting members. This is to ensure broad participation as well as to avoid voting deadlocks.
+A TSC member must attend two of the previous five meetings to be considered a "voting member" and be eligible to vote. Should a TSC member miss three or more of the prior five meetings, they will no longer vote or be counted toward the total number of voting members. This is to ensure broad participation as well as to avoid voting deadlocks.
 
-A voting member may regain their status by attending two consecutive meetings. They may participate in votes in the second meeting.
+A member may regain their voting status by attending two consecutive meetings. They may participate in votes in the second meeting.
 
-#### Resignation of voting members
+Elections of TSC members are exempted from this rule. All TSC members otherwise eligible to vote on a member election may do so regardless of their past attendance.
```

Comment:
Rather than “otherwise eligible” can we be explicit and say “whose seats are not up for re-election”? Are there other reasons a “TSC member” may not vote in the elections?

Reply:
For now, this is the only thing that would impact the ability to vote in something other than an election, and the main goal here was to ensure nobody votes on their own seat, so being explicit looks good to me too.